doc/conf.py

```diff
@@ -45,12 +45,21 @@
     "sphinx_reredirects",
 ]
 
+
+# Single file redirects are handled in this file and can be done by a pylint
+# contributor. See:
 # https://documatt.gitlab.io/sphinx-reredirects/usage.html
+# Directory redirects are handled in ReadTheDoc admin interface and can only be done
+# by pylint maintainers at the following URL:
+# https://readthedocs.org/dashboard/pylint/redirects/
 redirects: dict[str, str] = {
     # "<source>": "<target>"
     "intro": "index.html",
     "support": "contact.html",
     "user_guide/ide-integration": "installation.html",
+    "messages/messages_introduction": "user_guide/messages/index.html",
+    "messages/messages_list": "user_guide/messages/messages_overview.html",
```

Comment:
```suggestion
```

https://pylint--6628.org.readthedocs.build/en/6628/messages/messages_introduction

Doesn't work 😢 

Let's handle all the `messages/*` redirects in `RTD`. I'll fix them after this gets merged!

Reply:
Ha right, RTD seems to preempt the single file redirect. Let me add a comment.

Reply:
[350bf76](https://github.com/PyCQA/pylint/pull/6628/commits/350bf76ceea668af36c45a707e615aee43b9c126)

Reply:
Doesn't it mean we should do 
```
    "user_guide/messages/messages_introduction": "user_guide/messages/index.html",
    "user_guide/messages/messages_list": "user_guide/messages/messages_overview.html",
```    
?

Reply:
Lol, yeah that's probably it. Because the above link still doesn't work.

Reply:
Waiting for the pipeline to confirm it works but I think it make more sense to keep RTD for full dir redirect and our conf for single link redirect even if it means having two redirections instead of one.

Reply:
Hmm no, not working, even when clearing the redirect cache. Should I revert the latest commit ?

Reply:
The link we used to test is wrong I think. ``https://pylint--6628.org.readthedocs.build/en/6628/messages/messages_introduction`` should be ``https://pylint--6628.org.readthedocs.build/en/6628/messages/messages_introduction.html``